expyre/func.py

```diff
@@ -136,7 +141,7 @@ def __init__(self, name, *, input_files=[], env_vars=[], pre_run_commands=[], po
             # NOTE: following loop will not match status == 'processed' because such jobs are
             # not guaranteed to have results available.  Is it a good idea to try to use those,
             # if results actually seem to be available?
-            for job in config.db.jobs(status='can_produce_results', id=f'{name}_{arghash}_.*'):
+            for job in config.db.jobs(status='can_produce_results', id=re.escape(f'{name}_{arghash}_.*')):
```

Comment:
`.*` shouldn't be escaped, that never matches any of the already existing jobs. `id=re.escape(f'{name}_{arghash}_.')+'.*'`?

Reply:
Should be fixed in commit 8de5e46, which I forgot to push earlier, like the other things above.  @gelzinyte thanks for catching those - I found some myself, but not all.  Is it OK now?

Reply:
Thanks! Yes, nothing else has popped up.